src/EditorFeatures/CSharpTest/InlineDeclaration/CSharpInlineDeclarationTests_FixAllTests.cs

```diff
@@ -94,6 +95,53 @@ void M()
     {
         GetExeAndArguments(useCmdShell, executable, arguments, out string finalExecutable, out string finalArguments);
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineDeclaration)]
+        [WorkItem(28323, "https://github.com/dotnet/roslyn/issues/28323")]
+        public async Task FixAllInDocument4()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        long {|FixAllInDocument:ip|}; int port;
+        long.TryParse(v, out ip);
+        int.TryParse(v, out port);
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        long.TryParse(v, out long ip);
+        int.TryParse(v, out int port);
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineDeclaration)]
+        public async Task FixAllInDocument5()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        int {|FixAllInDocument:ip|}, port;
```

Comment:
let's have tests with 3 locals on the same line, where either hte first/second, first/third, or second/third get removed.